Project.toml

```diff
@@ -17,8 +17,8 @@ Scratch = "6c6a2e73-6563-6170-7368-637461726353"
 [compat]
 ColorTypes = "0.9, 0.10, 0.11, 0.12"
 Downloads = "1.3"
-FFMPEG = "0.3, 0.4"
-FFMPEG_jll = "4.1"
+FFMPEG = "0.3, 0.4, 1"
+FFMPEG_jll = "6.1"
```

Comment:
Do we still need to directly depend on both?

Reply:
https://github.com/JuliaIO/VideoIO.jl/pull/418#issuecomment-1950211326